content/docs/intro/cloud-providers/consul/setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: HashiCorp Consul Setup
+meta_desc: This page provides an overview on how to configure the Pulumi Consul Provider.
+---
+
+The [Pulumi Consul provider]({{< relref "./" >}}) uses the Consul SDK to manage resources.
+
+> Pulumi relies on the Consul SDK to authenticate requests from your computer to HashiCorp Consul. Your credentials are never sent
+> to pulumi.com.
+
+### Configuring The Provider
+
+Once obtained, there are two ways to communicate your configuration tokens to Pulumi:
+
+1. Set the environment variable `CONSUL_HTTP_ADDR`:
+
+    ```bash
+    $ export CONSUL_HTTP_ADDR=XXXXXX
+    ```
+
+2. Set them using configuration, if you prefer that they be stored alongside your Pulumi stack for easy multi-user access:
```

Comment:
Similarly "them" is plural, but we're only showing a single config value.

Reply:
Fixed